docs/pages/versions/unversioned/sdk/calendar.md

```diff
@@ -17,7 +17,9 @@ Provides an API for interacting with the device's system calendars, events, remi
 
 ## Configuration
 
-In managed apps, `Calendar` requires `Permissions.CALENDAR`. Interacting with reminders on iOS requires `Permissions.REMINDERS`.
+In managed apps you must add following permissions to your `app.json`.
```

Comment:
```suggestion
You must add following permissions to your `app.json`.
```
we can configure these permissions for you in bare as well 😄

Reply:
Good to know, I've only been working in managed work flow so don't pay too much attention to the dark side :)